059_API_L22/059_API_L22_unsafe.js

```
@@ -0,0 +1,339 @@
+Router.post('/login', (req, res) => {
+    const { auth } = req.body;
+    const { id, password } = auth;
+    console.log('login request', id, password);
+    attemptLogin(auth).then( result => {
+        //successfull login, send auth token to user
+        const { title, member_id, family_name, given_names, country_code, accommodation } = result;
+        pool.query(`SELECT CASE WHEN (SELECT EXISTS(SELECT * FROM olympics.Athlete A WHERE M.member_id = A.member_id)) = 't' THEN 'Athlete'
+	            WHEN (SELECT EXISTS(SELECT * FROM olympics.official O WHERE M.member_id = O.member_id)) = 't' THEN 'Official'
+	            WHEN (SELECT EXISTS(SELECT * FROM olympics.Staff S WHERE M.member_id = S.member_id)) = 't' THEN 'Staff' ELSE 'Unknown' END 
+                FROM olympics.member M WHERE M.member_id = $1;`,[member_id],  (err, response) =>{
+            if(err => console.log(err));
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)